recipes/miniaudio/all/test_package/example.c

```diff
@@ -0,0 +1,51 @@
+/*
+Demonstrates how to enumerate over devices.
+Device enumaration requires a `ma_context` object which is initialized with
+`ma_context_init()`. Conceptually, the context sits above a device. You can have
+many devices to one context. If you use device enumeration, you should
+explicitly specify the same context you used for enumeration in the call to
+`ma_device_init()` when you initialize your devices.
+*/
+#define MINIAUDIO_IMPLEMENTATION
+#include <miniaudio/miniaudio.h>
```

Comment:
All your examples, are including your lib as `<miniaudio.h>`, but in Conan recipe you're adding `include/` subfolder.

Reply:
As I commented, I did it because of the weird "extra" directory. I'm not sure having a stray "extra" directory is exactly sane. But if the goal is to be faithful to the library creator, whatever he did, I can remove it. What do you reckon?

Reply:
Removed it in 5e66103.